### ISSUE TITLE
Can I make panoptic occupancy map with your code?

### ISSUE THREAD

**Issue by joonsu0109gh** [weiyithu/SurroundOcc#24]

**Comment by weiyithu** [comment]
Hi, you mean the panoptic occupancy labels or using the panoptic images as inputs?

**Comment by joonsu0109gh** [comment]
I mean panoptic occupancy labels. 

Thank you for the prompt response.

**Comment by weiyithu** [comment]
Yeah, our generated labels are panoptic occupancy labels, not monocular ones.

**Comment by joonsu0109gh** [comment]
What I want to ask is whether I can add instance labels to the occupancy using NuScenes panoptic LiDAR segmentation labels.
I apologize for any confusion.

**Comment by lqzhao** [comment]
Yes, you can add instance labels to the foreground objects according to the known bounding boxes. Each bounding box has a unique 'object token', which can be the key for you to distinguish different objects with the same semantic labels.

**Comment by joonsu0109gh** [comment]
Do you mean that I can add instance labels with a bounding box without using LiDAR panoptic segmentation labels?

**Comment by lqzhao** [comment]
Hi, I'm not very familiar with "LiDAR panoptic segmentation labels" in nuScenes. If nuScenes already provide point-wise "LiDAR panoptic segmentation labels", you can use them to replace the LiDAR semantic segmentation labels which are used by our implementation.

The bounding boxes are used in our generation pipeline to crop movable objects from the scene. If nuScenes doesn't provide point-wise "LiDAR panoptic segmentation labels", you can use 'object token' to generate instance-level occupancy labels.
